test/routes/liana.collection.test.js

```diff
@@ -0,0 +1,274 @@
+describe('liana > collection', () => {
+  let forestExpress;
+  let spy;
+
+  const resetRequireIndex = () => {
+    jest.resetModules();
+    jest.restoreAllMocks();
+
+    // eslint-disable-next-line global-require
+    const logger = require('../../src/services/logger');
+    spy = jest.spyOn(logger, 'warn');
+
+    // eslint-disable-next-line global-require
+    forestExpress = require('../../src');
```

Comment:
Maybe adding a comment on that part would help understand why this is needed. I'm using a similar piece of code in my PR (#520) - maybe we should refactor that part once one of these is merged.

(Unrelated to your contribution) Also, we should find a way to properly mock injected services in the `src/index.js` in order to avoid the jest `spyOn` directly mocking the dependency